cookbooks/bcpc-hadoop/recipes/maven_config.rb

```diff
@@ -57,12 +57,17 @@
   custom_certs.map do |cert|
     execute "create keystore #{::File.basename(cert)}" do
       command <<-EOH
-        yes | keytool -v -alias #{::File.basename(cert)} -import \
-        -file #{cert} \
+        keytool -alias #{::File.basename(cert)} -list -file #{cert} \
         -keystore #{node['bcpc']['hadoop']['java_https_keystore']} \
-        -storepass changeit \
-        -trustcacerts \
-        -import
+        -storepass changeit
+        if [[ $? != 0 ]]; then
```

Comment:
can you use built-in chef guard clauses instead? https://docs.chef.io/resource_common.html#guards

Reply:
good point, PR updated.